frontend/src/metabase/search/utils/search-location/search-location.ts

```diff
@@ -6,9 +6,12 @@ import type {
 } from "metabase/search/types";
 import { SearchFilterKeys } from "metabase/search/constants";
 
-export function isSearchPageLocation(location: SearchAwareLocation): boolean {
-  const components = location.pathname.split("/");
-  return components[components.length - 1] === "search";
+export function isSearchPageLocation(location?: SearchAwareLocation): boolean {
+  if (location) {
+    const components = location.pathname.split("/");
+    return components[components.length - 1] === "search";
```

Comment:
Are there any other URLs that end with `/search` that we need to take care of?
If not, we could/should just check for `location.pathname === "/search"`.

Reply:
Yeah, it makes sense considering its usage - I didn't look at this since the code was already there before I arrived but definitely it's overengineered for this case. I'll reduce it to a boolean statement